xapian-letor/scorer/ndcg_score.cc

```diff
@@ -0,0 +1,71 @@
+/** @file ndcg_score.cc
+ *  @brief Implementation of NDCGScore
+ */
+/* Copyright (C) 2016 Ayush Tomar
```

Comment:
I missed v-hasu on this one. 
It uses a different version of DCG calculation, but its not entirely fresh.